docs/octant-plugin-installation.md

```diff
@@ -91,18 +91,22 @@ You can follow the steps listed below to install octant and antrea-octant-plugin
     export KUBECONFIG=/etc/kubernetes/admin.conf
     ```
 
-3. Build antrea-octant-plugin.
+3. Get corresponding antrea-octant-plugin binary from release/assets based on your environment.
 
+    For example, you can get antrea-octant-plugin-linux-x86_64 if it meets your operating system  and architecture.
```

Comment:
```suggestion
    For example, you can get antrea-octant-plugin-linux-x86_64 if it meets your operating system and architecture.
```

Reply:
+1 for this suggestion, but also replace "meets" with "matches"